.codeclimate.yml

```diff
@@ -5,3 +5,7 @@ checks:
   method-lines:
     config:
       threshold: 80
+  similar-code:
```

Comment:
Great work @wasibani-roy, What are these lines meant to do?

Reply:
This line handles sections that are similar in look but have different implementations. So code climate will not dwell to much on them.

Reply:
okay Roy